docs/quickstart.ipynb

```diff
@@ -350,6 +537,159 @@
+      "| C(group_id)[T.15.0] |      0.469 |        0.398 |     1.179 |      0.239 | -0.312 |   1.249 |\n",
+      "| C(group_id)[T.16.0] |     -0.135 |        0.396 |    -0.340 |      0.734 | -0.913 |   0.643 |\n",
+      "| C(group_id)[T.17.0] |     -0.005 |        0.401 |    -0.013 |      0.989 | -0.792 |   0.781 |\n",
+      "| C(group_id)[T.18.0] |      0.283 |        0.403 |     0.702 |      0.483 | -0.508 |   1.074 |\n",
+      "---\n",
+      "RMSE: 2.141 R2: 0.137 \n"
+     ]
+    }
+   ],
+   "source": [
+    "fit_dummy = pf.feols(\"Y ~ X1 + C(group_id) \", data=data, vcov=\"HC1\")\n",
+    "\n",
+    "fit_dummy.summary()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This is does not scale well! Imagine you have 1000 different levels of `group_id`. You would need to add 1000 dummy variables to your model. This is where fixed effect models come in handy. They allow you to control for these fixed effects without adding all these dummy variables. The way to do it ys by a *demeaning procedure*. The idea is to subtract the average value of each level of `group_id` from the respective observations. This way, we control for the fixed effects without adding all these dummy variables. Let's try to do this manually:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "###\n",
+      "\n",
+      "Estimation:  OLS\n",
+      "Dep. var.: Y_demeaned, Fixed effects: \n",
+      "Inference:  HC1\n",
+      "Observations:  998\n",
+      "\n",
+      "| Coefficient   |   Estimate |   Std. Error |   t value |   Pr(>|t|) |   2.5% |   97.5% |\n",
+      "|:--------------|-----------:|-------------:|----------:|-----------:|-------:|--------:|\n",
+      "| Intercept     |      0.003 |        0.068 |     0.041 |      0.968 | -0.130 |   0.136 |\n",
+      "| X1_demeaned   |     -1.019 |        0.083 |   -12.345 |      0.000 | -1.181 |  -0.857 |\n",
+      "---\n",
+      "RMSE: 2.141 R2: 0.126 \n"
+     ]
+    }
+   ],
+   "source": [
+    "def _demean_column(df: pd.DataFrame, column: str, by: str) -> pd.Series:\n",
+    "    return df[column] - df.groupby(by)[column].transform(\"mean\")\n",
+    "\n",
+    "\n",
+    "fit_demeaned = pf.feols(\n",
+    "    fml=\"Y_demeaned ~ X1_demeaned\",\n",
+    "    data=data.assign(\n",
+    "        Y_demeaned=lambda df: _demean_column(df, \"Y\", \"group_id\"),\n",
+    "        X1_demeaned=lambda df: _demean_column(df, \"X1\", \"group_id\"),\n",
+    "    ),\n",
+    "    vcov=\"HC1\",\n",
+    ")\n",
+    "\n",
+    "fit_demeaned.summary()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We the same results as the fixed effect model `Y1 ~ X | group_id` above. The `PyFixest` package uses a more efficient algorithm to estimate the fixed effect model, but the intuition is the same."
```

Comment:
> We the same
we get the

Reply:
Fixed in https://github.com/s3alfisc/pyfixest/pull/436/commits/c860669a015b0bb5e6541dacacb8a342fa7adee0